openapi/integrator/token-requestor/partner.yaml

```diff
@@ -176,16 +176,39 @@ components:
         * Accepted - Enrolment was accepted.
         * Rejected - Enrolment was rejected.
         * Failed - Enrolment failed.
-        * InvalidFundingPan - No eligible card found.
-        * CardExpired - Card has expired.
-        * CardNotActivated - Card is not activated.
       enum:
         - Accepted
         - Rejected
         - Failed
-        - InvalidFundingPan
-        - CardExpired
+
+    ErrorCode:
```

Comment:
New entity `ErrorCode` has been introduced, but not referenced in the doc yet.. Is it by design / what's the plan?
Removal of `InvalidFundingPan` and `CardExpired` from the original enum - is it ok/safe from the backward compatibility point of view during transition period?

Reply:
See internal comment.